Dockerfile

```diff
@@ -13,7 +13,7 @@ COPY cmd/ cmd/
 COPY internal/ internal/
 
 # Build
-RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 GO111MODULE=on go build -a -o manager ./cmd/manager/main.go
+RUN CGO_ENABLED=0 GO111MODULE=on go build -a -o manager ./cmd/manager/main.go
```

Comment:
Do we need to add these two env vars somewhere else? To the makefile, perhaps?

Reply:
I think it's okay if we're not explicit about OS and ARCH in the Dockerfile.